examples/decoding/plot_decoding_ssd_csp_eeg.py

```diff
@@ -0,0 +1,256 @@
+"""
+.. _ex-decoding-ssd-csp-eeg:
+
+============================================================================
+Improving motor imagery decoding from EEG using Spatio Spectra Decomposition
+============================================================================
+
+Improving the decoding of Motor Imagery from EEG data when SSD is applied
+before the traditional CSP. A classifier is then trained using the extracted
+features from the SSD+CSP-filtered signals.
+
+:footcite:`NikulinEtAl2011`.
+"""
+# Authors: Victoria Peterson <victoriapeterson09@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.pipeline import Pipeline
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis
+from sklearn.model_selection import ShuffleSplit, cross_val_score
+
+from mne import (Epochs, pick_types, events_from_annotations,
+                 EvokedArray)
+from mne.channels import make_standard_montage
+from mne.io import concatenate_raws, read_raw_edf
+from mne.datasets import eegbci
+from mne.decoding import CSP
+from mne.decoding import SSD
+
+###############################################################################
+# Set parameters and read data
+
+# avoid classification of evoked responses by using epochs that start 1s after
+# cue onset.
```

Comment:
Not sure I understand this, is cue onset at -2 seconds? Then maybe mention this (you cut from -1. to 4. below).

Reply:
Oh ! thank you, that shouldn't be there! I cropped the signal between 1 and 2 secs after the visual cue.